### ISSUE TITLE
I briked my minibook 

### ISSUE THREAD

**Issue by wokawoka** [juanjsebgarcia/chuwi-minibook-unlocked-bios#1]
I briked my minibook by flashing the original bios. Any help on how to flash the bios externally with an SOIC8/SOP8 adapter clip (I have it)?
Des anyone have a good BIOS dump?
Thank you

**Comment by DRCRecoveryData** [comment]
> I briked my minibook by flashing the original bios. Any help on how to flash the bios externally with an SOIC8/SOP8 adapter clip (I have it)?
> 
> Des anyone have a good BIOS dump?
> 
> Thank you

yes, i can dump via xgecu pro for you

**Comment by wokawoka** [comment]
Thank you very much for your reply.
I remember that I had the right bios image but I wasn't able to properly interface with the small chip without desoldering it (it's very tiny).
The flasher was establishing a connection but later it was losing it.
Do you have any suggestion about that?
Thank you

**Comment by DRCRecoveryData** [comment]
> Thank you very much for your reply.
> I remember that I had the right bios image but I wasn't able to properly interface with the small chip without desoldering it (it's very tiny).
> The flasher was establishing a connection but later it was losing it.
> Do you have any suggestion about that?
> Thank you
> 
You can use ch341a with clipper without sorldering it

**Comment by wokawoka** [comment]
> > Thank you very much for your reply.
> > I remember that I had the right bios image but I wasn't able to properly interface with the small chip without desoldering it (it's very tiny).
> > The flasher was establishing a connection but later it was losing it.
> > Do you have any suggestion about that?
> > Thank you
> 
> You can use ch341a with clipper without sorldering it

yes, I tried with the clipper but I remember that the I could not keep the clipper in place and the connection was not stable.
did you already try to read/write that chip on that laptop?
maybe I was doing something wrong

**Comment by DRCRecoveryData** [comment]
> > > Thank you very much for your reply.
> 
> > > I remember that I had the right bios image but I wasn't able to properly interface with the small chip without desoldering it (it's very tiny).
> 
> > > The flasher was establishing a connection but later it was losing it.
> 
> > > Do you have any suggestion about that?
> 
> > > Thank you
> 
> > 
> 
> > You can use ch341a with clipper without sorldering it
> 
> 
> 
> yes, I tried with the clipper but I remember that the I could not keep the clipper in place and the connection was not stable.
> 
> did you already try to read/write that chip on that laptop?
> 
> maybe I was doing something wrong
> 
> 

bios&ec link:
https://www.mediafire.com/file_premium/jtu1zygm0h6n9pn/BIOS_009%2526EC_08.rar/file

**Comment by wokawoka** [comment]
> > > > Thank you very much for your reply.
> > 
> > 
> > > > I remember that I had the right bios image but I wasn't able to properly interface with the small chip without desoldering it (it's very tiny).
> > 
> > 
> > > > The flasher was establishing a connection but later it was losing it.
> > 
> > 
> > > > Do you have any suggestion about that?
> > 
> > 
> > > > Thank you
> > 
> > 
> > > 
> > 
> > 
> > > You can use ch341a with clipper without sorldering it
> > 
> > 
> > yes, I tried with the clipper but I remember that the I could not keep the clipper in place and the connection was not stable.
> > did you already try to read/write that chip on that laptop?
> > maybe I was doing something wrong
> 
> bios&ec link: https://www.mediafire.com/file_premium/jtu1zygm0h6n9pn/BIOS_009%2526EC_08.rar/file

are you sure that it's possible to use the clippers to flash the eprom?
I rememeber that when I was trying the connection was not stable.
Do you know any other ways other than desoldering the chip?
Idea: as an alternative, would you be able to flash a chip and sell it to me so that I could bring it to a technician to swap it?

**Comment by DRCRecoveryData** [comment]
> > > > > Thank you very much for your reply.
> > > 
> > > 
> > > > > I remember that I had the right bios image but I wasn't able to properly interface with the small chip without desoldering it (it's very tiny).
> > > 
> > > 
> > > > > The flasher was establishing a connection but later it was losing it.
> > > 
> > > 
> > > > > Do you have any suggestion about that?
> > > 
> > > 
> > > > > Thank you
> > > 
> > > 
> > > > 
> > > 
> > > 
> > > > You can use ch341a with clipper without sorldering it
> > > 
> > > 
> > > yes, I tried with the clipper but I remember that the I could not keep the clipper in place and the connection was not stable.
> > > did you already try to read/write that chip on that laptop?
> > > maybe I was doing something wrong
> > 
> > 
> > bios&ec link: https://www.mediafire.com/file_premium/jtu1zygm0h6n9pn/BIOS_009%2526EC_08.rar/file
> 
> are you sure that it's possible to use the clippers to flash the eprom? I rememeber that when I was trying the connection was not stable. Do you know any other ways other than desoldering the chip? Idea: as an alternative, would you be able to flash a chip and sell it to me so that I could bring it to a technician to swap it?

which cilipper and programmer you used?

**Comment by wokawoka** [comment]
> > > > > > Thank you very much for your reply.
> > > > 
> > > > 
> > > > > > I remember that I had the right bios image but I wasn't able to properly interface with the small chip without desoldering it (it's very tiny).
> > > > 
> > > > 
> > > > > > The flasher was establishing a connection but later it was losing it.
> > > > 
> > > > 
> > > > > > Do you have any suggestion about that?
> > > > 
> > > > 
> > > > > > Thank you
> > > > 
> > > > 
> > > > > 
> > > > 
> > > > 
> > > > > You can use ch341a with clipper without sorldering it
> > > > 
> > > > 
> > > > yes, I tried with the clipper but I remember that the I could not keep the clipper in place and the connection was not stable.
> > > > did you already try to read/write that chip on that laptop?
> > > > maybe I was doing something wrong
> > > 
> > > 
> > > bios&ec link: https://www.mediafire.com/file_premium/jtu1zygm0h6n9pn/BIOS_009%2526EC_08.rar/file
> > 
> > 
> > are you sure that it's possible to use the clippers to flash the eprom? I rememeber that when I was trying the connection was not stable. Do you know any other ways other than desoldering the chip? Idea: as an alternative, would you be able to flash a chip and sell it to me so that I could bring it to a technician to swap it?
> 
> which cilipper and programmer you used?

sorry @DRCRecoveryData , I completely missed the notification of your message last month.
I believe I used a programmer like [this one](https://www.amazon.com/AiTrip-EEPROM-Programmer-CH341A-Adapter/dp/B07VNVVXW6) and the black clippers were not firlmly attaching to the pins of the smd epromm.
I remember I did try multiple times to read and write to the chip but at a certain point it was always failing, I guess that the connection was not firm.

Now I temporarily moved to another country and the (bricked) minibook and the programmer are not with me at the moment. But I'll be back soon next month.

What would you suggest to do? Ask a technician to desolder/resolder the eprom maybe? buy another kind of clipper?

Thank you very much!

**Comment by DRCRecoveryData** [comment]
> > > > > > > Thank you very much for your reply.
> 
> > > > > 
> 
> > > > > 
> 
> > > > > > > I remember that I had the right bios image but I wasn't able to properly interface with the small chip without desoldering it (it's very tiny).
> 
> > > > > 
> 
> > > > > 
> 
> > > > > > > The flasher was establishing a connection but later it was losing it.
> 
> > > > > 
> 
> > > > > 
> 
> > > > > > > Do you have any suggestion about that?
> 
> > > > > 
> 
> > > > > 
> 
> > > > > > > Thank you
> 
> > > > > 
> 
> > > > > 
> 
> > > > > > 
> 
> > > > > 
> 
> > > > > 
> 
> > > > > > You can use ch341a with clipper without sorldering it
> 
> > > > > 
> 
> > > > > 
> 
> > > > > yes, I tried with the clipper but I remember that the I could not keep the clipper in place and the connection was not stable.
> 
> > > > > did you already try to read/write that chip on that laptop?
> 
> > > > > maybe I was doing something wrong
> 
> > > > 
> 
> > > > 
> 
> > > > bios&ec link: https://www.mediafire.com/file_premium/jtu1zygm0h6n9pn/BIOS_009%2526EC_08.rar/file
> 
> > > 
> 
> > > 
> 
> > > are you sure that it's possible to use the clippers to flash the eprom? I rememeber that when I was trying the connection was not stable. Do you know any other ways other than desoldering the chip? Idea: as an alternative, would you be able to flash a chip and sell it to me so that I could bring it to a technician to swap it?
> 
> > 
> 
> > which cilipper and programmer you used?
> 
> 
> 
> sorry @DRCRecoveryData , I completely missed the notification of your message last month.
> 
> I believe I used a programmer like [this one](https://www.amazon.com/AiTrip-EEPROM-Programmer-CH341A-Adapter/dp/B07VNVVXW6) and the black clippers were not firlmly attaching to the pins of the smd epromm.
> 
> I remember I did try multiple times to read and write to the chip but at a certain point it was always failing, I guess that the connection was not firm.
> 
> 
> 
> Now I temporarily moved to another country and the (bricked) minibook and the programmer are not with me at the moment. But I'll be back soon next month.
> 
> 
> 
> What would you suggest to do? Ask a technician to desolder/resolder the eprom maybe? buy another kind of clipper?
> 
> 
> 
> Thank you very much!
> 
> 
> 
> 
> 
> What would you suggest?

Here’s a revised version:

1. You need to desolder the BIOS EEPROM and place it in a 200mil socket because the clip may have poor contact.

![image](https://github.com/user-attachments/assets/c24ac5dd-b16d-4724-96a8-5c997b77667e)

2. The software you need to use is Neo Programmer:

[Download Neo Programmer 2.2.0.10](https://www.mediafire.com/file/ianla0jca5z3ltb/NeoProgrammer_2.2.0.10.rar/file)

**Comment by wokawoka** [comment]
Thank you so much for your reply! I have very basic soldering skills and I'm afraid I'm not confident enough with SMD soldering. I'm afraid I could damage the board desoldering it and I believe I would not be able to solder back the small chip. That's why the minibook is briked and not being used since many years now. (CHUWI support was HORRIBLE and never replied to me despite the bad flash was from them).

Maybe I could buy the 200mil socket above and buy an extra chip and bring the programmed chip to a local electronic repair shop?

Do you think it could make sense?

Thank you very much again!!!

**Comment by DRCRecoveryData** [comment]
> Thank you so much for your reply! I have very basic soldering skills and I'm afraid I'm not confident enough with SMD soldering. I'm afraid I could damage the board desoldering it and I believe I would not be able to solder back the small chip. That's why the minibook is briked and not being used since many years now. (CHUWI support was HORRIBLE and never replied to me despite the bad flash was from them).
> 
> 
> 
> Maybe I could buy the 200mil socket above and buy an extra chip and bring the programmed chip to a local electronic repair shop?
> 
> 
> 
> Do you think it could make sense?
> 
> 
> 
> Thank you very much again!!!

Yes if you don't have iron heat you can ask electronic or mobile phone repair store to desolder the bios chip ~ 380 celcius temp

**Comment by wokawoka** [comment]
Thank you very much once again for all your suggestions!
I will do that as soon as I will be back in my country (in a few months) and report the results here.

**Comment by DRCRecoveryData** [comment]
> Thank you very much once again for all your suggestions!
> 
> I will do that as soon as I will be back in my country (in a few months) and report the results here.

Wanna share with you i have replacement the error keyboard to new US keyboard

![image](https://github.com/user-attachments/assets/7b188c97-291b-4159-92eb-fa5fb498c340)
![image](https://github.com/user-attachments/assets/19586cdd-c604-4fc1-9c12-855e5c213655)Wa

**Comment by wokawoka** [comment]
Oh, very nice
where did you find the replacement keyboard?
ebay?

**Comment by DRCRecoveryData** [comment]
> Oh, very nice where did you find the replacement keyboard? ebay?

I buy it from china factory

**Comment by wokawoka** [comment]
directly from chuwi's factory?
or aliexpress/taobao?

**Comment by DRCRecoveryData** [comment]
> directly from chuwi's factory?
> 
> or aliexpress/taobao?

in taobao:

![image](https://github.com/user-attachments/assets/739cbbe6-7de9-4013-98c4-9bb0d44a8cbb)

**Comment by wokawoka** [comment]
> > directly from chuwi's factory?
> > or aliexpress/taobao?
> 
> in taobao

oh, nice, thank you so much for sharing it!
